src/createNumbersGenerator.js

```diff
@@ -26,7 +26,16 @@
  * @return {number}
  */
 function createNumbersGenerator(numbers, min, max) {
-  // write code here
+  let count = 0;
+  const num = arguments.length === 3
```

Comment:
why does the length has to be 3?

Reply:
create default values instead
(https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters)